README.md

```diff
@@ -52,7 +51,20 @@ specified](#using-doctoc-to-generate-links-compatible-with-other-sites).
 
 ### Update existing doctoc TOCs effortlessly
 
-If you already have a TOC inserted by doctoc, it will automatically be updated by running the command (rather than inserting a duplicate toc). Doctoc locates the TOC by the `<!-- START doctoc -->` and `<!-- END doctoc -->` comments, so you can also move a generated TOC to any other portion of your document and it will be updated there.
+If you already have a TOC inserted by doctoc, it will automatically be updated 
+by running the command (rather than inserting a duplicate toc).
+Doctoc locates the TOC by the `<!-- START doctoc -->` and `<!-- END doctoc -->` comments, 
+so you can also move a generated TOC to any other portion of your document and it will be updated there.
+
+When passing a directory to doctoc you can tell it to **only update** files that **already** have a TOC 
+(including those that [specify location of toc](#specifying-location-of-toc)),
+by adding the `--explicit` flag:
+
+    doctoc . --explicit
+
+When being ***explicit*** by passing one ore more file arguments to doctoc,
```

Comment:
Not a huge fan of that _feature_.
If I pass you only files then I still expect the ones to be affected that already have a TOC since I passed the explicit flag.
You never know where those filenames come from, i.e. could be from a `*.md` pattern expanded in another script.

So please **no magic** ;)